packages/debug/src/browser/style/index.css

```diff
@@ -54,6 +54,11 @@
 .theia-source-breakpoint .path {
     font-size: var(--theia-ui-font-size0);
     color: var(--theia-descriptionForeground);
```

Comment:
@akosyakov My guess is that maybe the css selector in above is more specific than [#7359 (comment)](https://github.com/eclipse-theia/theia/pull/7359#issuecomment-601900513)